integration/tests/epochs/recover_epoch/suite.go

```diff
@@ -12,10 +24,85 @@ type Suite struct {
 func (s *Suite) SetupTest() {
 	// use a shorter staking auction because we don't have staking operations in this case
 	s.StakingAuctionLen = 2
-	s.DKGPhaseLen = 50
-	s.EpochLen = 250
+	s.DKGPhaseLen = 10
+	s.EpochLen = 80
 	s.EpochCommitSafetyThreshold = 20
 
 	// run the generic setup, which starts up the network
 	s.BaseSuite.SetupTest()
 }
+
+// getNodeInfoDirs returns the internal node private info dir and the node config dir from a container with the specified role.
+func (s *Suite) getNodeInfoDirs(role flow.Role) (string, string) {
+	internalNodePrivInfoDir := fmt.Sprintf("%s/%s", s.GetContainersByRole(role)[0].BootstrapPath(), bootstrap.DirPrivateRoot)
+	nodeConfigJson := fmt.Sprintf("%s/%s", s.GetContainersByRole(role)[0].BootstrapPath(), bootstrap.PathNodeInfosPub)
+	return internalNodePrivInfoDir, nodeConfigJson
+}
+
+// executeEFMRecoverTXArgsCMD executes the efm-recover-tx-args CLI command to generate EpochRecover transaction arguments.
+// Args:
+//
+//	role: the container role that will be used to read internal node private info and the node config json.
+//	snapshot: the protocol state snapshot.
+//	collectionClusters: the number of collector clusters.
+//	numViewsInEpoch: the number of views in the recovery epoch.
+//	numViewsInStakingAuction: the number of views in the staking auction of the recovery epoch.
+//	epochCounter: the epoch counter.
+//	targetDuration: the target duration for the recover epoch.
+//	targetEndTime: the target end time for the recover epoch.
+//	randomSource: the random source of the recover epoch.
+//	out: the tx args output file full path.
```

Comment:
Looks like this list is outdated, could you update it?

Reply:
https://github.com/onflow/flow-go/pull/6156/commits/78fb541350443ecf695db8b3536eefceec076fbd